data-otservbr-global/scripts/quests/soul_war/action-reward_soul_war.lua

```diff
@@ -37,7 +37,7 @@ function phantasmalJadeMount.onUse(player, item, fromPosition, target, toPositio
 	end
 
 	if table.contains({ 34072, 34073, 34074 }, item.itemid) then
-		if player:getItemCount(34072) >= 4 and player:getItemCount(34073) == 1 and player:getItemCount(34074) == 1 then
+		if player:getItemCount(34072) >= 4 and player:getItemCount(34073) >= 1 and player:getItemCount(34074) >= 1 then
```

Comment:
Isn't that better?

`if player:removeItem(34072, 4) and player:removeItem(34073, 1) and player:removeItem(34074, 1) then`

Reply:
I think that this is not a good idea, because with this code that you are suggesting it could lead to the removal of the first 4 items but if the others items did not fulfill the requirement of 1 item each, it will not deliver the mount and it will be removing the items that was fulfilled.

Reply:
Either way, the mechanics are completely wrong. Here is how it works correctly:
https://www.youtube.com/watch?v=qkZQiHLUvjA